sky/serve/serve_utils.py

```diff
@@ -23,6 +23,7 @@
 from sky import exceptions
 from sky import global_user_state
 from sky import status_lib
+from sky.backends import backend_utils
```

Comment:
Importing `backend_utils` here may caues circular import. We should be careful about this.